getinsomnia/insomnia
# feat: enable ProxyConfig in pre-request scripting [INS-3379]

Changes:
- [x] enable ProxyConfig in pre-request scripting

### How to test:

Please run this script and check its output.
```
const { ProxyConfig } = require('insomnia-collection');

const proxyConfig = new ProxyConfig({
		match: 'http+https://*.example.com:80/*',
		host: 'proxy.com',
		port: 8080,
		tunnel: true,
		disabled: false,
		authenticate: true,
		username: 'proxy_username',
		password: 'proxy_password',
});

console.log(proxyConfig.getProxyUrl());
```
